### DISCUSSION TITLE
Project Folders / Group repos

### DISCUSSION THREAD

**Discussion by athyk** [community/community#8443] [Other Feature Feedback, Questions, & Ideas] [answered]
At least now if I make anything that is related is either having it as a different branch or prefix it with the same thing. Creating a whole organisation is very unneeded. I don't know if there is a way of doing similar already.

My thought is to have a collapsible folders that you can see in the repositories tab.

Also thinking of allowing it to be starred and added to pinned boards.

Then you can assign members go these folders without doing every repo. Organisations work for this but sometimes you want to give more control on certain projects then others.

**Comment by BellettiR** [comment, marked as answer]
There's a similar discussion here: https://github.com/github/feedback/discussions/4174
I think it's something similar to what you're asking!

**Comment by MarcoValliIMA** [comment]
There's also a similar discussion here: #5387
